libs/gi/ui/src/components/artifact/editor/SubstatInput.tsx

```diff
@@ -129,26 +128,31 @@ export function SubstatInput({
                 </MenuItem>
               ))}
           </DropdownButton>
-          <CustomNumberInputButtonGroupWrapper
-            sx={{ flexBasis: 30, flexGrow: 1 }}
+          <Box
+            sx={{
+              flexBasis: 30,
+              flexGrow: 1,
+              padding: 0,
+              div: {
+                width: '100%',
+                height: '100%',
+              },
+            }}
           >
-            <CustomNumberInput
+            <NumberInputLazy
               float={unit === '%'}
               placeholder={t`editor.substat.selectSub`}
-              value={key ? value : undefined}
+              value={key ? value : 0}
               onChange={(value) =>
                 setSubstat(index, { key, value: value ?? 0 })
               }
               disabled={!key}
               error={!!error}
-              sx={{
-                px: 1,
-              }}
               inputProps={{
-                sx: { textAlign: 'right' },
+                sx: { textAlign: 'right', height: '1vh' },
```

Comment:
What does the `1vh` here do?

Reply:
was trying to set the size to be what it was before, moved it to the parent Box